backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/internal/BpkDialogImpl.kt

```diff
@@ -98,29 +96,60 @@ internal fun BpkFlareDialogImpl(
   }
 }
 
+@Composable
+internal fun BpkImageDialogImpl(
+  onDismissRequest: () -> Unit,
+  title: String,
+  text: String,
+  buttons: List<Dialog.Button>,
+  properties: DialogProperties,
+  textAlign: TextAlign,
+  content: @Composable BoxScope.() -> Unit,
+) {
+  Dialog(onDismissRequest = onDismissRequest, properties = properties) {
+    Surface(
+      shape = BpkTheme.shapes.medium,
+      color = BpkTheme.colors.surfaceDefault,
+    ) {
+      Column {
+        Box(content = content)
+        DialogContent(title = title, text = text, textAlign = textAlign, buttons = buttons)
+      }
+    }
+  }
+}
+
 @Composable
 private fun DialogContent(title: String, text: String, buttons: List<Dialog.Button>, modifier: Modifier = Modifier) {
```

Comment:
I'd also inline this function, it's just one parameter specification. You can set it ti be Center by default or just specify every time